### PR TITLE
Add NeuVector chart version 2.7.3 to dev v2.8

### PR DESCRIPTION
**Pull request by selvamt94**
## Problem
NeuVector has released app version 5.3.0 along with helm chart 2.7.3, which need to be added to Rancher feature charts version 2.8 (release-v2.8). We would like to get this chart released in the next OOB.

## Solution
new chart has been added

### Manual Testing
Tested fresh install of the NeuVector chart 103.0.2+up2.7.3 and upgraded from previous version on RKE and RKE2 cluster.

## QA Testing Considerations
Tested fresh install of the NeuVector chart 103.0.2+up2.7.3 and upgraded from previous version on RKE and RKE2 cluster.

### CODE REVIEW

**Comment by andsont** [comment]
both https://github.com/rancher/charts/pull/3463 and https://github.com/rancher/charts/pull/3464 are reviewed.
all of conditions for image are removed in all deployment yaml files except the registry-adapter.yaml and exporter-deployment.yaml. wondering if u want to make it consistent with other files? if the condition for image is not a concern for rancher chart, it looks good to me.
```
          {{- if .Values.oem }}
          image: "{{ .Values.registry }}/{{ .Values.oem }}/registry-adapter:{{ .Values.cve.adapter.image.tag }}"
          {{- else }}
          image: "{{ .Values.registry }}/registry-adapter:{{ .Values.cve.adapter.image.tag }}"
          {{- end }}
          {{- else }}
          {{- if .Values.cve.adapter.image.hash }}
          image: "{{ .Values.registry }}/{{ .Values.cve.adapter.image.repository }}@{{ .Values.cve.adapter.image.hash }}"
          {{- else }}
          image: {{ template "system_default_registry" . }}{{ .Values.cve.adapter.image.repository }}:{{ .Values.cve.adapter.image.tag }}
          {{- end }}
          {{- end }}
```

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

File: release.yaml [hunk 1]
```diff
@@ -58,11 +58,17 @@ rancher-vsphere-cpi:
   - 103.1.0+up1.7.0
 rancher-vsphere-csi:
   - 103.1.0+up3.1.2-rancher1
+neuvector:
```
Comment (lucasmlp) [inline release.yaml]: There is another set of entries for neuvector and neuvector-crd in line 8-10

**Review by lucasmlp — CHANGES_REQUESTED** [review]
See release.yaml

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

**Pull request merged**